arch/Sparc/SparcInstPrinter.c

```diff
@@ -168,6 +168,9 @@ static void printOperand(MCInst *MI, int opNum, SStream *O)
 	unsigned reg;
 	MCOperand *MO = MCInst_getOperand(MI, opNum);
 
+	if (opNum >= MI->size) {
+		return;
+	}
```

Comment:
is this necessary, or only fixing MCInst.c is enough?

Reply:
yes fixing MCInst.c is enough, and is more relevant than this previous fix : the same bug came with other architectures...